src/components/scene/inspector.js

```diff
@@ -0,0 +1,52 @@
+/* global AFRAME */
+var registerComponent = require('../../core/component').registerComponent;
+var pkg = require('../../../package');
+
+var INSPECTOR_URL = pkg.homepage + 'releases/' + pkg.version + '/aframe-inspector.min.js';
+
+module.exports.Component = registerComponent('inspector', {
+  schema: {
+    url: {default: INSPECTOR_URL}
+  },
+
+  init: function () {
+    this.onKeydown = this.onKeydown.bind(this);
+  },
+
+  play: function () {
+    window.addEventListener('keydown', this.onKeydown);
+  },
+
+  pause: function () {
+    window.removeEventListener('keydown', this.onKeydown);
+  },
+
+  onKeydown: function (evt) {
+    // Alt + Ctrl + e
+    var shortcutPressed = evt.keyCode === 69 && evt.ctrlKey && evt.altKey;
```

Comment:
in the case of embedded scenes, the `<iframe>` can't listen to any keypresses until you first click it. on page load, the `document.activeElement` will be `<body>` of the parent window.

and, it's probably rare, but the keyboard shortcut might get accidentally triggered if a user typed it while typing in a form field (e.g., a `<textarea>` on a Wikipedia page or a comment field of a page with embedded scenes made with A-Frame).

also, what happens on the page when I have two embedded scenes? we probably don't want both listeners to fire?